wheatley/row_generation/place_notation_generator.py

```diff
@@ -48,6 +50,10 @@ def parse_call_dict(unparsed_calls: CallDef) -> Dict[int, List[Places]]:
 
         self._generating_call_pn: List[Places] = []
 
+    def summary_string(self) -> str:
+        """ Returns a short string summarising the RowGenerator. """
+        return f"place notation '{self.method_pn_string}'"
```

Comment:
I guess this will only be seen for Grandsire and Stedman for the moment?

Reply:
Yep.  We could probably provide a name, but when we convert to using Bob Wallis' blueline website for methods, then we won't need the custom methods anyway.